### PR TITLE
Fixes for retinanet - annotation path mismatch

### PR DESCRIPTION
**Pull request by anandhu-eng**

### CODE REVIEW

File: script/get-dataset-openimages/customize.py [hunk 1]
```diff
@@ -69,7 +69,6 @@ def postprocess(i):
         env['CM_DATASET_PATH'] = os.path.join(os.getcwd(), 'install', 'validation', 'data')
         annotations_file_path = os.path.join(env['CM_DATASET_ANNOTATIONS_DIR_PATH'], "openimages-mlperf.json")
         env['CM_DATASET_VALIDATION_ANNOTATIONS_FILE_PATH'] = annotations_file_path
-        env['CM_DATASET_OPENIMAGES_VALIDATION_ANNOTATIONS_FILE_PATH'] = annotations_file_path
```
Comment (arjunsuresh) [inline script/get-dataset-openimages/customize.py:72]: I think this is useful to have

File: script/get-dataset-openimages/_cm.json [hunk 2]
```diff
@@ -79,7 +79,7 @@
     "CM_CALIBRATION_DATASET_PATH",
     "CM_CALIBRATION_DATASET_PATH_ROOT",
     "CM_OPENIMAGES_CALIBRATION_DATASET_PATH",
-    "CM_DATASET_OPENIMAGES_VALIDATION_ANNOTATIONS_FILE_PATH"
```
Comment (arjunsuresh) [inline script/get-dataset-openimages/_cm.json:82]: We can keep this too

Comment (anandhu-eng) [reply script/get-dataset-openimages/_cm.json:82]: Updated in https://github.com/GATEOverflow/cm4mlops/pull/153/commits/077b930155ceffd16f2304f20c74f26fe34aa02a.

File: script/get-dataset-openimages/customize.py (hunk 1 above)
Comment (anandhu-eng) [reply script/get-dataset-openimages/customize.py:72]: Updated in https://github.com/GATEOverflow/cm4mlops/pull/153/commits/c9bbd2a814e78b7ee93215752e2d7f099a056486

**Review by arjunsuresh — APPROVED** [review]

**Pull request merged**